test/math_test.dart

```diff
@@ -40,22 +27,40 @@ class MathTestModule extends Module {
     final aTimesB = addOutput('a_times_b', width: a.width);
     final aDividedByB = addOutput('a_dividedby_b', width: a.width);
     final aModuloB = addOutput('a_modulo_b', width: a.width);
+    final aModuleConst = addOutput('a_modulo_const', width: a.width);
```

Comment:
typo: `Module` should be `Module`, no?

Reply:
Yes, but you made the same typo :)

Thanks for the catch!

Reply:
Created #209